apps/studio.giselles.ai/drizzle/schema.ts

```diff
@@ -246,3 +247,24 @@ export const agentTimeRestrictions = pgTable(
 		teamDbIdIdx: index().on(table.teamDbId),
 	}),
 );
+
+export const invitations = pgTable(
+	"invitations",
+	{
+		token: text("token").notNull().unique(),
+		teamDbId: integer("team_db_id")
+			.notNull()
+			.references(() => teams.dbId, { onDelete: "cascade" }),
+		email: text("email").notNull(),
+		role: text("role").notNull().$type<TeamRole>(),
+		inviteUserDbId: integer("invite_user_db_id")
```

Comment:
```suggestion
		inviterUserDbId: integer("inviter_user_db_id")
```

Reply:
Nits:

Since the nit comment concerns English parts of speech, I'll leave it to you to make the revisions.